Service/FilterPathContainer.php

```diff
@@ -0,0 +1,83 @@
+<?php
+declare(strict_types=1);
```

Comment:
so far we are not declaring strict types explicitly in other places of the code.

Reply:
But the declaring strict types is here.
https://github.com/liip/LiipImagineBundle/blob/master/Imagine/Cache/Helper/PathHelper.php#L3

Reply:
ah, then we just don't do it consistently. all good then.

guess someone could then clean this up on a followup.